### PR TITLE
Unittests for postgres

### PR DESCRIPTION
**Pull request by pontus**
Adds test coverage for `internal/postgres`.

### CODE REVIEW

File: internal/postgres/postgres_test.go [hunk 1]
```diff
@@ -0,0 +1,636 @@
...
+		diSet := map[string][]string{"dataset1": accessions[0:3],
+			"dataset2": accessions[4:5],
+			"dataset3": accessions[6:8],
+			"dataset4": accessions[9:9]}
+
+		success := sqlmock.NewResult(1, 1)
+
+		for di, acs := range diSet {
+			mock.ExpectBegin()
+			for _, aId := range acs {
```
Comment (jbygdell) [inline internal/postgres/postgres_test.go:552]:
```suggestion
			for _, aID := range acs {
```

File: internal/postgres/postgres_test.go [hunk 2]
```diff
@@ -0,0 +1,636 @@
...
+				r := sqlmock.NewRows([]string{"file_id"})
+
+				fileId, _ := strconv.Atoi(aId)
```
Comment (jbygdell) [inline internal/postgres/postgres_test.go:555]:
```suggestion
				fileID, _ := strconv.Atoi(aID)
```

File: internal/postgres/postgres_test.go [hunk 3]
```diff
@@ -0,0 +1,636 @@
...
+
+				r.AddRow(fileId)
```
Comment (jbygdell) [inline internal/postgres/postgres_test.go:557]:
```suggestion
				r.AddRow(fileID)
```

File: internal/postgres/postgres_test.go [hunk 4]
```diff
@@ -0,0 +1,636 @@
...
+
+				mock.ExpectQuery("SELECT file_id FROM local_ega.archive_files WHERE stable_id = \\$1").
+					WithArgs(aId).WillReturnRows(r)
```
Comment (jbygdell) [inline internal/postgres/postgres_test.go:560]:
```suggestion
					WithArgs(aID).WillReturnRows(r)
```

File: internal/postgres/postgres_test.go [hunk 5]
```diff
@@ -0,0 +1,636 @@
...
+
+				mock.ExpectExec("INSERT INTO local_ega_ebi.filedataset \\(file_id, dataset_stable_id\\) VALUES \\(\\$1, \\$2\\);").
+					WithArgs(fileId, di).WillReturnResult(success)
```
Comment (jbygdell) [inline internal/postgres/postgres_test.go:563]:
```suggestion
					WithArgs(fileID, di).WillReturnResult(success)
```

**Comment by pontus** [comment]
I consider testify/assert but was hesitant to add new dependencies, redone with it now.

**Pull request merged**